Document built-in System.Net activities (#43872)
dotnet/docs: docs/core/diagnostics/built-in-metrics-system-net.md

| `http.request.method` | string | HTTP request method. | `GET`; `POST`; `HEAD`; `_OTHER` [2] | Always |
| `http.response.status_code` | int | [HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6). | `200` | If response was received. |
| `network.protocol.version` | string | Version of the HTTP protocol used. | `1.1`; `2` | If response was received. |
| `server.address` | string | Host identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. | `example.com` | Always |
| `server.port` | int | Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. | `80`; `8080`; `443` | Depends on .NET version. [3] |
| `url.scheme` | string | The [URI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1) component identifying the used protocol. | `http`; `https`; `ftp` | Always |

**[1] `error.type`:** If the request has failed, the value is set to one of the following:

- An exception name with type, e.g., `System.Threading.Tasks.TaskCanceledException`

> - A status code that indicates a client or server error, e.g., `500`

Reviewer: ```suggestion
- A status code that indicates a client or server error, for example, `500`.
```

> - In case an <xref:System.Net.Http.HttpRequestException> occurred and with a <xref:System.Net.Http.HttpRequestError> other than `Unknown`; the enum value in snake case, e.g., `name_resolution_error`

Reviewer: ```suggestion
- If an <xref:System.Net.Http.HttpRequestException> occurred with an <xref:System.Net.Http.HttpRequestError> other than `Unknown`, the enum value in snake case, for example, `name_resolution_error`.
```
